ILRepack/ILRepack.cs

```diff
@@ -352,7 +352,9 @@ public void Repack()
                 Logger.Info("Output directory does not exist. Creating output directory: " + outputDir);
                 Directory.CreateDirectory(outputDir);
             }
-            TargetAssemblyDefinition.Write(Options.OutputFile, parameters);
+            using (FileStream outputStream = File.OpenWrite(Options.OutputFile))
+                TargetAssemblyDefinition.Write(outputStream, parameters);
```

Comment:
@gluck You linked `ModuleDefinition`, but this is on `AssemblyDefinition` which is different: https://github.com/gluck/cecil/blob/c854cad1a1da59ea94e5e87cbcc0b0a1416c9b62/Mono.Cecil/AssemblyDefinition.cs#L157